lib/hexpm/accounts/totp.ex

```diff
@@ -0,0 +1,124 @@
+defmodule Hexpm.Accounts.TOTP do
+  alias Hexpm.Accounts.BackupCode
+
+  @app_secret Application.get_env(:hexpm, :totp_encryption_secret)
```

Comment:
Avoid putting config in module attributes unless it's necessary. It's because changing config requires recompilation when in attributes.